### PR TITLE
updata Chinese

### PR DESCRIPTION
**Pull request by xinjiajuan**

### CODE REVIEW

File: res/values-zh-rCN/yaap_strings.xml [hunk 1]
```diff
@@ -175,4 +182,29 @@
...
+    <string name="about_dev2_name">Ido Ben-Hur</string>
+    <string name="about_dev2_git" translatable="false">https://github.com/idoybh</string>
+    <string name="about_dev3_title">会员</string>
+    <string name="about_dev3_name">Omkar Chandorkar</string>
+    <string name="about_dev3_git" translatable="false">https://github.com/gotenksIN</string>
+    <string name="maintainer">设备维护者</string>
+    <string name="maintainer_name" translatable="false"></string>
+    <string name="maintainer_telegram" translatable="false"></string>
+    <string name="about_tg_title">电报群</string>
+    <string name="about_tg_summary">全球电报集团官方</string>
```
Comment (ALKALiKong233) [inline res/values-zh-rCN/yaap_strings.xml]: *社区* not *集团*
Are you using Google translate LMAO 🤣 @ZONGQUAN-HU

**Comment by idoybh** [comment]
Yet again. your commits are a mess. Lemme fix that

**Review by ALKALiKong233 — CHANGES_REQUESTED** [review]
`<string name="about_tg_summary">官方国际Telegram群组</string>`

File: res/values-zh-rCN/yaap_strings.xml (hunk 1 above)
Comment (idoybh) [reply res/values-zh-rCN/yaap_strings.xml]: AHAHAHAHAHAHAHA this is hysterical

**Comment by xinjiajuan** [comment]
> 再次。你的提交是一团糟。解决问题
Please point out the problem.  I did not translate the text that was not allowed to be translated, but left it in the file.

File: res/values-zh-rCN/yaap_strings.xml (hunk 1 above)
Comment (idoybh) [reply res/values-zh-rCN/yaap_strings.xml]: @ALKALiKong233 can you suggest the change so I can commit it? What exactly should be the string?

**Comment by idoybh** [comment]
> > 再次。你的提交是一团糟。解决问题
> > Please point out the problem.  I did not translate the text that was not allowed to be translated, but left it in the file.

The problem is in git as always..... Always fetch latest changes *before* you do your changes

File: res/values-zh-rCN/yaap_strings.xml (hunk 1 above)
Comment (xinjiajuan) [reply res/values-zh-rCN/yaap_strings.xml]: 帮忙改一下吧，把集团改成社区，集团 change to 社区

**Comment by xinjiajuan** [comment]
> > > 再次。你的提交是一团糟。解决问题
> > > Please point out the problem.  I did not translate the text that was not allowed to be translated, but left it in the file.
> 
> The problem is in git as always..... Always fetch latest changes _before_ you do your changes

But I did not find the problem, the submitted file is also this one

**Comment by idoybh** [comment]
> But I did not find the problem, the submitted file is also this one

Because I force-pushed it out just now. There is no problem anymore except the string @ALKALiKong233 reported. But the problem was not in the file. It was in commit history

**Review by idoybh — APPROVED** [review]

**Pull request merged**